src/apps/reservation-list/modal/delete-reservation/delete-reservation-modal.tsx

```diff
@@ -65,6 +67,7 @@ const DeleteReservationModal: FC<DeleteReservationModalProps> = ({
         )
       );
 
+      close(reservationsQueued as string);
```

Comment:
I am a bit unsure if this is too closely coupled?
Another option would be to pass in the underlying `modalId` we want closed with itself. 
Thoughts?